utils/download_and_sync_courses.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import logging
+import os
+import zipfile
+from typing import Dict, Any
+from urllib.parse import urlparse, urljoin
+import omegaup.api
+import shutil
+import http.client
+import ssl
+
+# Create SSL context that skips certificate verification
+context = ssl._create_unverified_context()
+
+logging.basicConfig(level=logging.INFO)
+LOG = logging.getLogger(__name__)
+
+API_CLIENT = None
+BASE_URL = None
+
+# 👇 Add your course aliases here
+COURSE_ALIASES = [
+    "omi-public-course"
+]
+
+BASE_COURSE_FOLDER = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "Courses"))
+
+
+if os.path.exists(BASE_COURSE_FOLDER):
+    LOG.warning("Delete existing course folder to avoid conflicts")
+    shutil.rmtree(BASE_COURSE_FOLDER)
+
+
+
+
```

Comment:
With the help of `pycodestyle`, we'll avoid adding unnecessary blank lines. But I think those changes are in a different PR.